### PR TITLE
Ag 6327/column sparkline

### PR DESCRIPTION
**Pull request by manapeirov**

### CODE REVIEW

File: enterprise-modules/sparklines/src/sparkline/bar-column/barSparkline.ts [hunk 1]
```diff
@@ -80,7 +87,7 @@ export class BarSparkline extends BarColumnSparkline {
             const height =
                 xScale instanceof BandScale
                     ? xScale.bandwidth
-                    : Math.abs(yScale.range[1] - yScale.range[0]) / data.length;
```
Comment (alantreadway) [inline enterprise-modules/sparklines/src/sparkline/bar-column/barSparkline.ts:83]: Guessing this was the source of the weird rendering we were seeing last week?

File: enterprise-modules/sparklines/src/sparkline/sparkline.ts [hunk 2]
```diff
@@ -514,17 +514,23 @@ export abstract class Sparkline extends Observable {
...
                 return;
             }
             const distance = this.getDistance(hitPoint, datum.point);
-            if (distance < minDistance) {
+            if (distance <= minDistance) {
                 minDistance = distance;
                 closestDatum = datum;
             }
-        });
+            else {
```
Comment (alantreadway) [inline enterprise-modules/sparklines/src/sparkline/sparkline.ts:530]: Nit: Remove leading newline?

File: enterprise-modules/sparklines/src/sparkline/bar-column/barSparkline.ts (hunk 1 above)
Comment (manapeirov) [reply enterprise-modules/sparklines/src/sparkline/bar-column/barSparkline.ts:83]: Which weird rendering do you mean

File: enterprise-modules/sparklines/src/sparkline/sparkline.ts (hunk 2 above)
Comment (manapeirov) [reply enterprise-modules/sparklines/src/sparkline/sparkline.ts:530]: oh yeah dunno how that happened

Comment (manapeirov) [reply enterprise-modules/sparklines/src/sparkline/sparkline.ts]: done

**Pull request merged**